integrations/mixpanel/.buildscript/deploy_snapshot.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
This file shouldn't be required in the monorepo. This supported a manual release process, which is now deprecated.